assets/src/components/list-items/index.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+export function ListItems( { className = '', heading, items } ) {
+	const slugify = ( value ) => {
+		if ( 'string' !== typeof value ) {
+			return value;
+		}
+
+		return value
+			.toString()
+			.trim()
+			.toLowerCase()
+			.replace( /\s+/g, '-' )
+			.replace( /[^\w\-]+/g, '-' )
+			.replace( /\-\-+/g, '-' )
+			.replace( /^-+/, '-' )
+			.replace( /-+$/, '-' );
+	};
```

Comment:
I don't think we need this function. According to the [SO answer:](https://stackoverflow.com/a/28984464):

> The `key` just needs to be a basic type that can be treated as a hash value to determine uniqueness. So, a string or a number for example.

Since in our case we always have at least a `label` or a `value`, I think that the key could be something like:

```js
key={ `${ item.value }${ item.label }` }
```

I think this should always be unique in our case.

What's more, we shouldn't include the `index` value from the `map` function in the `key` since we're back in the same position as before (the reason why we wanted to change the `key` value was to get rid of the `index` value).

Reply:
> I don't think we need this function. According to the SO answer::

We can remove the function name,

However, we have to keep using `index`. Since there are some cases when we don't pass the label and the value is JSXElement (e.g While rendering Validate URLs). and for the same reason, we won't be able to use value.